scripts/tutorials/single_spectra_example.py

```diff
@@ -0,0 +1,42 @@
+# Example adding a single spectrum to the database
+
+from astrodbkit2.astrodb import Database
+from datetime import datetime
+
+connection_string = 'sqlite:///SIMPLE.db'  # SQLite
+db = Database(connection_string)
+
+# Add missing telescopes, etc
+db.Telescopes.insert().execute([{'name': 'IRTF'}])
+db.Instruments.insert().execute([{'name': 'SpeX'}])
+db.Modes.insert().execute([{'name': 'Prism',
+                            'instrument': 'SpeX',
+                            'telescope': 'IRTF'}])
+
+# Add actual Spectra
+spec_data = [{'source': '2MASS J00192626+4614078',
+              'regime': 'infrared',
+              'spectrum': 'https://s3.amazonaws.com/bdnyc/SpeX/Prism/U10013_SpeX.fits',
+              'telescope': 'IRTF',
+              'instrument': 'SpeX',
+              'mode': 'Prism',
+              'reference': 'Cruz18',
+              'wavelength_units': 'um',
+              'flux_units': 'erg s-1 cm-2 A-1',
+              'observation_date': datetime.fromisoformat('2004-11-08')
+              }]
+db.Spectra.insert().execute(spec_data)
+
+_ = db.inventory('2MASS J00192626+4614078', pretty_print=True)
```

Comment:
perhaps better variable names than the unnamed default "_"? (also, I can't see this being used?)

Reply:
The unnamed variable _ is a throwaway. This is just because db.inventory returns output that I don't want to see.